src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void greetings() {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void addList (Task[] array, String message, int itemNumber) {
+        Task taskItem = new Task(message);
+        array[itemNumber] = taskItem;
+        System.out.println("added: " + taskItem.description);
+    }
+
+    public static void addTodo (Task[] array, String message, int itemNumber) {
+        String[] splitMessage = message.split(" ", 2);
```

Comment:
Plural form should be used on names representing a collection of objects. Might want to rename it appropriately